ddp-workspace/projects/ddp-dsm-ui/src/app/add-patients-modal/add-patients-modal.component.html

```diff
@@ -0,0 +1,67 @@
+<p>Add Patient(s)</p>
+<p>
+  Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo itaque vel
+  labore quasi numquam sapiente quidem a corporis, nobis inventore veritatis
+  minus rem similique praesentium. Consequuntur, ab tenetur! Quis, ab.
+</p>
+
+<div class="patient-form">
+  <div class="row">
+    <div class="field">
+      <label>First Name</label>
+      <input
+        [value]="firstName$ | async"
```

Comment:
It's worth using [Angular Reactive Forms](https://angular.io/guide/reactive-forms) for all these inputs.
They have lots of benefits, including easy inputs validation.